properties/P000102.md

```diff
@@ -25,5 +25,4 @@ Equivalently:
 - There exists a symmetric $d$ such that $p$ lies in the closure of some $Y\subseteq X$ iff $d(p,Y)=\inf\{d(p,y):y\in Y\}=0$.
 Such a symmetric is automatically a semimetric.
 - $X$ is {P2}. And, for each $x \in X$, there is a non-increasing sequence of open sets $\{G_n(x):n<\omega\}$ forming a local neighborhood basis for $x$. Moreover, if for some sequence $x_n$ in $X$, $y\in X$ belongs to $G_n(x_n)$ for all $n$, then $x_n\to y$. In particular, $X$ is {P28}. This equivalence is shown in Theorem 3.2 of {{zb:0113.16501}}.
-
```

Comment:
Need to restore this blank line.  Otherwise, see the bad effect in preview mode for branch `pr1156`.